src/main/kotlin/org/malachite/estella/mails/Payloads.kt

```diff
@@ -110,3 +112,16 @@ fun HrPartner.toRegistrationMailPayload(password: String) =
             this.user.mail, password, MAIN_URL),
         sender_email = MAIN_MAIL
     )
+
+fun TaskStage.toTaskAssignmentRequestPayload(mail: String, offer: Offer): MailPayload {
+    val hrPartnerFullName = "${offer.creator.user.firstName} ${offer.creator.user.lastName}"
+    val url = "${MAIN_URL}task_stage/${this.id}/${offer.creator.organization.id}"
```

Comment:
Wydaje mi się, że ten url jest zły XDD
Zwłaszcza że ten widoczek musi mieć zakodowanego base64 maila, czego tutaj nie widać. Pamiętaj, że to nie jest link do RSa, tylko do Main Front Appa, a ja go pisałem jakoś tak:
mainURL/tasks/assign/{organizationUUID}/{devMailBase64}